languages/python/exercises/concept/string-methods/.docs/introduction.md

```diff
@@ -0,0 +1,16 @@
+## String Methods
```

Comment:
We've recently changed the spec a bit. Each concept this exercise unlocks should be a `## <concept-slug>` heading (although we will be changing this later on). Could you change this to:

```suggestion
## string-methods
```